packages/core/src/background/handlers/Tabs.ts

```diff
@@ -126,6 +126,14 @@ export default class Tabs {
     return this.#state.provideUid(url, request);
   }
 
+  private async isSet (): Promise<boolean> {
+    const uidRecords = await this.#state.allUidRecords();
+
+    const account = getSelectedIdentifiedAccount();
```

Comment:
I think we need to be clear when no account is found/selected, so that the app, and consequently the user, know what to do. Eg:

```
assert(account, 'No account is present or no account selected');
```